examples/03_connectivity/plot_data_driven_parcellations.py

```diff
@@ -188,6 +198,100 @@
 kmeans_labels_img.to_filename('kmeans_parcellation.nii.gz')
 
 ##################################################################
-# Finally show them
+# Brain parcellations with ReNA Clustering
+# ----------------------------------------
+# The spatial constraints are implemented inside the ReNA object.
+#
+# We use caching. As a result, the clustering doesn't have
+# to be recomputed later.
+from nilearn.regions import ReNA
+start = time.time()
+rena = ReNA(scaling=True, n_clusters=1000, memory='nilearn_cache')
+
+rena.fit_transform(dataset.func[0])
+print("ReNA 1000 clusters: %.2fs" % (time.time() - start))
```

Comment:
Let's only fit with 2000 clusters.

Reply:
Actually, given that ReNA is really beneficial only for a very large number of clusters, I would suggest to use 5000 clusters with ReNA.

Reply:
@GaelVaroquaux 
I believe we need to be consistent with what is done for ward. So, may be we could use n_clusters=5000 for ward also. Otherwise, it is hard to compare the computation time. Are you okay with that ?

Reply:
@bthirion

Reply:
No, I do not think that we need to be consistent with ward.